src/app/components/game-components/tablut/tablut.component.ts

```diff
@@ -9,19 +9,26 @@ import { LegalityStatus } from 'src/app/jscaip/LegalityStatus';
 import { TablutCase } from 'src/app/games/tablut/tablutrules/TablutCase';
 import { display } from 'src/app/collectionlib/utils';
 import { MGPValidation } from 'src/app/collectionlib/mgpvalidation/MGPValidation';
+import { Player } from 'src/app/jscaip/player/Player';
 
 @Component({
     selector: 'app-tablut',
-    templateUrl: './tablut.component.html',
+    templateUrl: './old-tablut.component.html',
```

Comment:
Yeup, there is two version now, I'm refactoring the way it's done to make it more like epaminondas (full svg, using angular way to make it smart) but since it's not finished yet, we use the old one.
Should I do a ticket about that ?